### DISCUSSION TITLE
BentoML 1.0

### DISCUSSION THREAD

**Discussion by callmedada** [bentoml/BentoML#2428] [Q&A]
Hey,

Just wondering if the 1.0 version is suggested to be used in the production environment given it is still in the pre-release stage. As I found it seems have better performance than 0.13. And if possible, what will be the release plan for the 1.0 version.

Thanks a million,
Dennis

**Comment by timliubentoml** [comment]
Hi Dennis! We're actively working on 1.0 as you've noticed. We been releasing new versions pretty frequently. I would say if a particular version is stable for you, then it's probably good to use, but between versions things can sometimes not be backwards compatible as we're iterating on the design quickly.

We plan to ship a release candidate version in the next month.

By the way, love to hear about real performance use cases. How did you benchmark and how much faster was it?

Thanks,

Tim
